README.md

```diff
@@ -1,14 +1,44 @@
 # Named Entity Recognition for chatbots
-Haptik is one of the world's largest conversational AI companies redefining the way chatbots work. 
 
-To build a good chatbot, one of the modules that play a crucial role is [Named Entity Recognition](https://en.wikipedia.org/wiki/Named-entity_recognition) (NER). In this repository, we have open-sourced our NER which uses heuristic approach along with several NLP techniques to extract necessary entities from chat interfaces.
+![chatbotner logo](docs/images/chatbotner_logo.png)
 
-To know more about our NER repository please go through the following documentations:
+Chatbot NER is an open source framework custom built to supports entity recognition in text messages. After doing
+thorough research on existing [NER](https://en.wikipedia.org/wiki/Named-entity_recognition) systems, team at Haptik felt
+the strong need of building a framework which is tailored for Conversational AI and also supports Indian languages.
+Currently Chatbot-ner supports **English, Hindi, Gujarati, Marathi, Bengali and Tamil** and their code mixed form.
+Currently this framework uses heuristics along with few NLP techniques to extract necessary entities from languages
+with sparse data. API structure of Chatbot ner is designed keeping in mind usability for Conversational AI
+applications. Team at Haptik is continuously working towards scaling this framework for **all Indian languages and their
```

Comment:
scaling should be replaced with porting throughout

Reply:
done